core/sail/api/src/main/java/org/eclipse/rdf4j/sail/helpers/AbstractSail.java

```diff
@@ -383,10 +386,18 @@ public long getIterationCacheSyncThreshold() {
 
 	/**
 	 * Set the threshold for syncing query evaluation iteration caches to disk.
-	 * 
+	 *
 	 * @param iterationCacheSyncThreshold The iterationCacheSyncThreshold to set.
 	 */
 	public void setIterationCacheSyncThreshold(long iterationCacheSyncThreshold) {
 		this.iterationCacheSyncThreshold = iterationCacheSyncThreshold;
 	}
+
+	public boolean isTrackResultSize() {
+		return trackResultSize;
+	}
+
+	public void setTrackResultSize(boolean trackResultSize) {
+		this.trackResultSize = trackResultSize;
+	}
```

Comment:
This is as high as it goes at the moment. Essentially for those who want to track performance for every query. Could also be useful if we ever implement a "slow query" log.